handlers/accessToken/handler.go

```diff
@@ -0,0 +1,44 @@
+package accessToken
+
+import (
+	"context"
+	"errors"
+	"net/http"
+
+	"github.com/ONSdigital/go-ns/common"
+	"github.com/ONSdigital/go-ns/log"
+)
+
+// CheckHeader is a wrapper which adds a accessToken from the request header to context if one does not yet exist
+func CheckHeader(h http.Handler) http.Handler {
```

Comment:
Is `CheckHeader` an accurate description of what this handler actually does? `ForwardAccessTokenIfExists` or something along those lines feels more appropriate (to me anyway).